### PR TITLE
Add java parser - fixes #473

### LINKED ISSUE #473: Java support with java-parser
Lately I've been building with [java-parser](https://github.com/jhipster/prettier-java/tree/e7648a257b9e1ba47a39e3d057d085ff6235427c/packages/java-parser), and I'd love to see support on astexplorer. Would you take a PR for this?

### PR DESCRIPTION
**Pull request by junoatscroll**
I followed the instructions in readme about adding a new parser- pretty painless! Please let me know if there are any changes you'd like or other review comments.

### CODE REVIEW

File: website/src/parsers/java/java-parser.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  locationProps: new Set(["location"]),
+  typeProps: new Set(["name"]),
+
+  loadParser(callback) {
+    require(["java-parser"], callback);
+  },
+
+  parse(parser, code) {
+    console.time("p");
```
Comment (fkling) [inline website/src/parsers/java/java-parser.js]: I'd prefer not to have these in here.

File: website/src/parsers/java/java-parser.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    const cst = parser.parse(code);
+
+    const clone = JSON.parse(JSON.stringify(cst));
```
Comment (fkling) [inline website/src/parsers/java/java-parser.js]: Why is this necessary?

**Comment by junoatscroll** [comment]
Thanks for reviewing this! Those lines you found were not supposed to be left in there, those were debugging lines. I'll clean up this PR soon when I have a moment.
